client/public/manifest.json

```diff
@@ -1,13 +1,6 @@
 {
   "short_name": "tackle-ui",
   "name": "Tackle UI",
-  "icons": [
```

Comment:
Do you know what this manifest.json is for? I imagine having the favicon in here is important for some reason. If we use the webpack config approach above I think we can restore these lines since I think it will just use `favicon.ico` as the compiled filename regardless of the source file.